dev-client/src/screens/AddUserToProjectScreen/components/AddTeamMemberForm.tsx

```diff
@@ -64,7 +67,10 @@ export const AddTeamMemberForm = ({projectId}: FormProps) => {
       // Success
       else {
         const user = userOrError as UserFields;
-        navigation.navigate('ADD_USER_PROJECT_ROLE', {projectId, user});
+        const userId = user.id;
+        // Currently there is no need to access any user's preferences besides the current user
+        dispatch(addUserToCache({...user, preferences: {}}));
```

Comment:
I'm thinking of making a tech debt item to address this weirdness -- I don't think the app should need to know any user preferences except for the current user... so perhaps the cached set of `users` in accountSlice should not include user preferences, and the current user's preferences should be stored in a different fashion?

Reply:
Talked with garo, will try to:
- change checkUserInProject to an async action so I can add the user there instead of here
- add tech debt items for not getting other users' preferences (and other user data that should potentially be private)